src/sidebar/store/modules/session.js

```diff
@@ -68,6 +77,8 @@ function isFeatureEnabled(state, feature) {
  * Return true if the user's profile has been fetched. This can be used to
  * distinguish the dummy profile returned by `profile()` on startup from a
  * logged-out user profile returned by the server.
+ *
+ * @param {object} state - The application state
```

Comment:
Can you remove the `state` param JSDoc for the moment. This changed yesterday such that `state` now refers to the local module state in most cases.

Reply:
👍